bin/deploy-vhost

```diff
@@ -339,10 +329,46 @@ sub update_vcspath_symlink {
     shell("ln -snf $vhost_dir/$vcspath_install $vhost_dir/$vcspath");
 }
 
+sub daemons {
+    my $type = shift;
+    my @daemons;
+    foreach my $daemon (keys %{$conf->{'daemons'}}) {
+        my $daemon_mugly_file;
+        my $servers;
+        my $config = $conf->{'daemons'}->{$daemon};
+        if (ref $config eq 'HASH') {
+            $daemon_mugly_file = $config->{file};
+            $servers = $config->{servers};
+            $servers = [ $servers ] if $servers && ref $servers ne 'ARRAY';
+        } else {
+            $daemon_mugly_file = $config;
+        }
+        if (!$servers && $config->{daemons_internal}) {
+            $servers = $conf->{servers_internal};
+        }
+
+        next if $type eq 'banish' && (!$servers || grep { m/^$hostname$/ } @$servers);
+        next if $type eq 'summon' && ($servers && !grep { m/^$hostname$/ } @$servers);
+
+        push @daemons, {
+            name => $daemon,
+            file => $daemon_mugly_file,
+            servers => $servers,
```

Comment:
Do we need to send this back? I don't think it's used anywhere else.

Reply:
Although it's referenced below on L389 I don't think it needs to be.

Reply:
Oh yes, think I used to do the server check outside the daemons() but then moved it inside, so yeah, doesn't need to be.